File: Java/primes.java
public class primes {
    public static void main(String[] args) {
        long start = System.currentTimeMillis();
        System.out.println("Started in Java");
        int i = 0;
        int primes = 0;
        while (i < 150000) {
            boolean isPrime = true;
            for (int j = 2; j < i; j++) {
                if (i % j == 0) {
                    isPrime = false;
                    break;
                }
            }
            if (isPrime) {
                primes++;
            }
            i++;
        }
        long end = System.currentTimeMillis();
        double time = (end - start) / 1000.0;
        System.out.println("Execution time: " + time + " seconds");
        System.out.println("Primes found: " + primes);
        System.out.println("Primes per second: " + (primes / time));
    }
}

File: Javascript/primes.js
let start = new Date().getTime()
console.log('Started in JavaScript')
let i = 0
let primes = 0
while (i < 150000) {
  let isPrime = true
  for (let j = 2; j < i; j++) {
    if (i % j === 0) {
      isPrime = false
      break
    }
  }
  if (isPrime) {
    primes++
  }
  i++
}
let end = new Date().getTime()
let time = end - start
console.log('Execution time: ' + time / 1000 + ' seconds')
console.log('Primes found: ' + primes)
console.log('Primes per second: ' + (primes / time * 1000))

